netlify/edge-functions/enhance.mts

```diff
@@ -1,21 +1,20 @@
-/*
-    Enhance the returned HTML by adding a link to the repository URL
-*/
-import { HTMLRewriter } from "https://ghuc.cc/worker-tools/html-rewriter/index.ts";
-import { Config, Context } from "@netlify/edge-functions";
-import data from "../data.json" assert { type: "json" };
+/**
+ * Enhance the returned HTML by adding a link to the repository URL
+ */
+import { HTMLRewriter } from 'https://ghuc.cc/worker-tools/html-rewriter/index.ts';
+import { Config, Context } from '@netlify/edge-functions';
+import data from '../data.json' with { type: 'json' };
```

Comment:
Apparently this is the way with Deno. I see a deprecation warning when using `assert`. However, Prettier fails with `with`. Not going to dig too far into that for now, since we don't really need folks messing with this function.